flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/sink/FileSinkTest.java

```diff
@@ -23,14 +23,14 @@
 import org.apache.flink.connector.file.sink.utils.PartSizeAndCheckpointRollingPolicy;
 import org.apache.flink.core.fs.Path;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
 
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for {@link FileSink}. */
-public class FileSinkTest {
+class FileSinkTest {
 
     @Test
     public void testCreateFileWriterWithTimerRegistered() throws IOException {
```

Comment:
```suggestion
    void testCreateFileWriterWithTimerRegistered() throws IOException {
```
as mentioned earlier, could be omitted